docs/support/faq/how-to-resize-in-pouta.md

```diff
@@ -6,12 +6,19 @@ It's possible to proceed by taking a snapshot of your instance and then boot the
 You can find more information about snapshot [here](../../cloud/pouta/snapshots.md) and how to proceed.
 
 ### Using the resize functionality 
+
+Few points regarding the resize functionality:
+
+- [I/O flavors](../../cloud/pouta/vm-flavors-and-billing.md#io-flavors_2) and [GPU flavors](../../cloud/pouta/vm-flavors-and-billing.md#gpu-flavors_2) cannot be resized to a different family flavors. I/O flavors use an ephemeral disk. The data that are not volumes are not copied when the resize functionality is used. This means that your machine won't be able to boot and the message "No bootable device" will be displayed in the console. We recommend to use [snapshots](../../cloud/pouta/snapshots.md#launching-a-snapshot-from-the-web-interface).
+
+- You can safely use the resize process if your I/O flavor vm is booted from a volume.
```

Comment:
this is such a corner case (using io flavors + booting from volume + not using ephemeral disk at all) that I would consider removing the sentence